### ISSUE TITLE
AddOnBefore publisher ignores "." in variable names

### ISSUE THREAD

**Issue by fvet** [DavidFeldhoff/al-codeactions#146]
When running the 'AddOnBeforePublisher' code action on below function ...

`local procedure ShowDetails(Type: enum "Purchase Line Type"; "No.": Code[20])`

... the "." is stripped from the variable name.

![image](https://user-images.githubusercontent.com/25268332/195267746-3bb703f8-f94f-4319-989e-3b390379f023.png)

Resulting in an invalid publisher call.

![image](https://user-images.githubusercontent.com/25268332/195267636-5af757cf-46c5-4317-842a-13d44cf41d32.png)

PS: Would be good if the publisher arguments would take over the original variable name (respecting the dot)

**Comment by DavidFeldhoff** [comment]
Well said :D I agree that it would be "nice" to have working code afterwards :D
Let me look into it

**Comment by DavidFeldhoff** [comment]
It should be fixed now with v1.0.24 :)

**Comment by fvet** [comment]
Tested and approved!
Thanks for the quick fix.

```
local procedure MyProcedure("No.": code[20]; "Field_": Code[20]; "Description/Name": code[20]; " Space": code[20])
    var
        IsHandled: Boolean;
    begin
        OnBeforeMyProcedure("No.", "Field_", "Description/Name", " Space", IsHandled);
        if IsHandled then
            exit;

        OnAfterMyProcedure("No.", "Field_", "Description/Name", " Space");
    end;
```
